### ISSUE TITLE
Excellent work!

### ISSUE THREAD

**Issue by raywzy** [tomguluson92/unsup3D_pytorch3d#1]
Have you figured out the reason of poor results of PyTorch3D?

**Comment by tomguluson92** [comment]
Nope, But I think this is due to the different rendering interface. I think one could get a reasonable result with dive more into the code of `unsup3D`. I think my version is just not enough,
